### PR TITLE
[KYUUBI #304] Remove abbreviations for service control arguments

### PR DESCRIPTION
**Pull request by turboFei**
turboFei [Closes #482](https://github.com/yaooqinn/kyuubi/pull/482)    [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _Why are the changes needed?_
Remove  abbreviations for service control arguments

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/KyuubiCtlArguments.scala [hunk 1]
```diff
@@ -128,7 +128,7 @@ class KyuubiCtlArguments(args: Seq[String], env: Map[String, String] = sys.env)
     val command = sys.env.getOrElse("_KYUUBI_CMD_USAGE",
```
Comment (yaooqinn) [inline kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/KyuubiCtlArguments.scala:128]: _KYUUBI_CMD_USAGE, is the first underscore a typo?

**Review by yaooqinn — APPROVED** [review]

File: kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/KyuubiCtlArguments.scala (hunk 1 above)
Comment (turboFei) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/KyuubiCtlArguments.scala:128]: emmm

**Comment by turboFei** [comment]
also cc @pan3793

File: kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/KyuubiCtlArgumentsSuite.scala [hunk 2]
```diff
@@ -49,7 +49,7 @@ class KyuubiCtlArgumentsSuite extends KyuubiFunSuite {
       Seq("server", "engine").foreach { service =>
         val args = Seq(
           command, service,
-          "--zkAddress", zkAddress,
+          "--zk-quorum", zkAddress,
```
Comment (pan3793) [inline kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/KyuubiCtlArgumentsSuite.scala:52]: should we also change the name of `zkAddress` and `ZK_ADDRESS`?

Comment (turboFei) [reply kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/KyuubiCtlArgumentsSuite.scala]: thanks

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thx, merged.

**Pull request closed without merging**